features/collection/shell-commands/src/main/java/org/opennms/netmgt/collection/commands/CollectCommand.java

```diff
@@ -146,7 +146,7 @@ public Void execute() {
         final CompletableFuture<CollectionSet> future = locationAwareCollectorClient.collect()
                 .withAgent(agent)
                 .withSystemId(systemId)
-                .withCollector(collector)
+                .withCollectorClassName(className)
```

Comment:
Is this change necessary? We already use the `ServiceCollectorRegistry` to lookup the collector above.

Reply:
I guess it relates to the fact that the CollectorRequestBuilderImpl ends up with the proxy class name instead, if that's it we should add a comment.